README.md

```diff
@@ -8,21 +8,32 @@ The various configuration frameworks are intialised by supplying appsetting.json
 # Example Usage
 
 ```
-[Binding]
+[   [Binding]
```

Comment:
Looks like a syntactic error here. Does this still build?!

Reply:
it's in the readme so doesn't impact the build.  but fixed :-)